docs/docs/50-user-guide/60-reference-docs/30-promotion-steps/jira.md

```diff
@@ -0,0 +1,651 @@
+  requestedFreight:
+    - origin:
+        kind: Warehouse
+        name: nginx
+      sources:
+        direct: true
+  promotionTemplate:
+    spec:
+      vars:
+        - name: imageRepo
+          value: public.ecr.aws/nginx/nginx
+      steps:
+        # Create initial deployment ticket
+        - as: create-deployment-ticket
+          uses: jira
+          config:
+            credentials:
+              secretName: jira
+            createIssue:
+              projectKey: KD
+              issueType: Task
+              summary: "Deploy Release ${{ imageFrom(vars.imageRepo).Tag }}"
+              assigneeEmail: "devops@company.com"
+              adfDescription:
+                type: doc
+                version: 1
+                content:
+                  - type: paragraph
+                    content:
+                      - type: text
+                        text: " "
+                  - type: heading
+                    attrs:
+                      level: 3
+                    content:
+                      - type: text
+                        text: "Automated deployment ticket for release "
+                      - type: text
+                        text: "${{ imageFrom(vars.imageRepo).Tag }}"
+                        marks:
+                          - type: code
+                  - type: paragraph
+                    content:
+                      - type: text
+                        text: "Image:"
+                        marks:
+                          - type: strong
+                      - type: text
+                        text: " "
+                      - type: text
+                        text: "${{ imageFrom(vars.imageRepo).RepoURL }}:${{ imageFrom(vars.imageRepo).Tag }}"
+                        marks:
+                          - type: code
+                  - type: paragraph
+                    content:
+                      - type: text
+                        text: "Project:"
+                        marks:
+                          - type: strong
+                      - type: text
+                        text: " "
+                      - type: text
+                        text: "${{ ctx.project }}"
+                        marks:
+                          - type: code
+              labels:
+                - "automated-deployment"
+                - "env-${{ ctx.stage.name }}"
+                - "release-${{ imageFrom(vars.imageRepo).Tag }}"
+                - "project-${{ ctx.project }}"
+
+        # Update application
+        - as: update-app
+          uses: argocd-update
+          config:
+            apps:
+              - name: test-app
+                namespace: argocd
+                sources:
+                  - repoURL: https://github.com/company/app-config.git
+                    kustomize:
+                      images:
+                        - repoURL: public.ecr.aws/nginx/nginx
+                          tag: ${{ imageFrom("public.ecr.aws/nginx/nginx").Tag }}
```

Comment:
This is [pseudo GitOps](https://docs.kargo.io/user-guide/patterns/#pseudo-gitops). I know it's an easy/accessible example, but if we're responsible about it, it probably needs some kind of disclaimer. It might be better to stick a `# ...` comment here to make it clear that other steps that weren't really relevant to the example were omitted.

May apply in other spots as well.